frontend/src/component/JobPostingGallery.tsx

```diff
@@ -0,0 +1,17 @@
+import {User} from "../model/User";
+import JobPostingCard from "./JobPostingCard";
+
+export type JobPostingGalleryProps = {
+    user: User
+}
+
+export default function JobPostingGallery(props: JobPostingGalleryProps) {
+
+
+    return (<div>
+        {props.user!==null&&props.user.jobPostings.length!==0&&props.user.jobPostings.map(
```

Comment:
Tipp: der Elvis-Operator "?." (sieht aus wie die Frisur von Elvis Presley) hilft Dir hier den Code zu kürzen.
```suggestion
        {props.user?.jobPostings.length!==0&&props.user.jobPostings.map(
```

Reply:
Danke..! Hab mich schon gefragt, wie ich das besser schreiben kann ☺️